### PR TITLE
Add PLplot

### PR DESCRIPTION
**Pull request by dsendkowski**
A cross-platform software package for creating scientific plots.

### CODE REVIEW

**Comment by pyhalov** [comment]
Fails to build, perhaps not safe to build in several threads (COMPONENT_BUILD_ARGS=-j4).
```
/bin/bash ../../libtool --tag=F77 --mode=link /usr/gcc/4.9/bin/gfortran -I/export/home/alp/srcs/tests/oi-userland/components/image/plplot/plplot-5.8.0/bindings/f77 -m64  -m64 -o libplplotf77d.la -rpath /usr/lib/amd64 -version-info 10:0:1 -rpath /usr/lib/amd64 -no-undefined libplplotf77cd.la strutil.lo sfstubs.lo configurable.lo  -lm 
libtool: link: cannot find the library `libplplotf77cd.la' or unhandled argument `libplplotf77cd.la'
make[4]: *** [Makefile:602: libplplotf77d.la] Error 1
```

**Comment by dsendkowski** [comment]
Is it mandatory? Should every component be built with this flag?

**Comment by pyhalov** [comment]
Well, you have 2 options:
1. fix original build system so that parallel build was not a problem;
2. add COMPONENT_BUILD_ARGS= in component Makefile.

**Comment by dsendkowski** [comment]
So adding `COMPONENT_BUILD_ARGS +=-j1` is a possible solution, right?

**Comment by pyhalov** [comment]
COMPONENT_BUILD_ARGS = 
is a more common idiom

**Comment by dsendkowski** [comment]
Ok, I added `COMPONENT_BUILD_ARGS`.

File: components/image/plplot/Makefile [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+COMPONENT_ARCHIVE_HASH=	sha256:595bcc021b415ba191979586543f95285f03f5b54ed77f452976a89ad2850877
+COMPONENT_ARCHIVE_URL=	http://sourceforge.net/projects/$(COMPONENT_NAME)/files/$(COMPONENT_NAME)/$(COMPONENT_VERSION)%20Source/$(COMPONENT_ARCHIVE)/download
+COMPONENT_LICENSE=	LGPLv2
+COMPONENT_LICENSE_FILE=	COPYING.LIB
+
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+COMPONENT_BUILD_ENV=	DEST_DIR=$(PROTO_DIR)
```
Comment (pyhalov) [inline components/image/plplot/Makefile]: Sorry, missed this. DEST_DIR is not necessary for this component.

File: components/image/plplot/patches/0-pltclgen.patch [hunk 2]
```diff
@@ -0,0 +1,14 @@
+--- plplot-5.8.0/bindings/tcl/pltclgen	Sun Nov 18 21:04:00 2007
```
Comment (pyhalov) [inline components/image/plplot/patches/0-pltclgen.patch]: style issue: we usually have patches starting from 01-*.patch

**Comment by dsendkowski** [comment]
Ok, corrected.

File: components/image/plplot/Makefile [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+COMPONENT_ARCHIVE_HASH=	sha256:595bcc021b415ba191979586543f95285f03f5b54ed77f452976a89ad2850877
+COMPONENT_ARCHIVE_URL=	http://sourceforge.net/projects/$(COMPONENT_NAME)/files/$(COMPONENT_NAME)/$(COMPONENT_VERSION)%20Source/$(COMPONENT_ARCHIVE)/download
+COMPONENT_LICENSE=	LGPLv2
+COMPONENT_LICENSE_FILE=	COPYING.LIB
+
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+COMPONENT_INSTALL_ENV=	DEST_DIR=$(PROTO_DIR)
```
Comment (pyhalov) [inline components/image/plplot/Makefile]: The same DEST_DIR...

**Comment by dsendkowski** [comment]
I hope this time it's fine.

**Pull request merged**